### PR TITLE
fix(console): generate user password

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

User password requires a minimum of 8 characters and contains a mix of letters, numbers, and symbols.

## Testing

Test locally.

## Checklist

- [ ] `.changset-staged`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/console/src/pages/Users/components/CreateForm/utils.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { customAlphabet } from 'nanoid';
+
+const alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+const passwordLength = 8;
+
+// Note: password requires a minimum of 8 characters and contains a mix of letters, numbers, and symbols.
+export const createPassword = () => {
```
Comment (gao-sun) [inline packages/console/src/pages/Users/components/CreateForm/utils.ts:7]:
```suggestion
export const createInitialPassword = () => {
```

File: packages/console/src/pages/Users/components/CreateForm/utils.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  const randomLength = Math.floor(Math.random() * (passwordLength - 1)) + 1;
```
Comment (gao-sun) [inline packages/console/src/pages/Users/components/CreateForm/utils.ts:8]: as it's an initial password, i think there's no need to random length

Comment (gao-sun) [reply packages/console/src/pages/Users/components/CreateForm/utils.ts:8]: and it'll be great to keep it simple, e.g. fixed one number, fixed one alphabet, and followed by six random number or alphabet chars

Comment (simeng-li) [reply packages/console/src/pages/Users/components/CreateForm/utils.ts:8]: + 1
${randomDigit}${randomAlphabet}${randomDigitOrAlphabet}(6)

File: packages/console/src/pages/Users/components/CreateForm/utils.ts (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/pages/Users/components/CreateForm/utils.ts]: updated.

File: packages/console/src/pages/Users/components/CreateForm/utils.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  const randomNumbers = Array.from({ length: randomLength }, () =>
+    Math.floor(Math.random() * 10)
+  ).join('');
+  const randomCharacters = customAlphabet(alphabet, passwordLength - randomLength)();
+
+  return `${randomNumbers}${randomCharacters}`
+    .split('')
+    .slice()
+    .sort(() => Math.random() - 0.5)
```
[the remark this answers is not in the document]
Comment (xiaoyijun) [reply packages/console/src/pages/Users/components/CreateForm/utils.ts]: removed

File: packages/console/src/pages/Users/components/CreateForm/utils.ts (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/pages/Users/components/CreateForm/utils.ts]: updated.

**Review by gao-sun — APPROVED** [review]

File: packages/console/src/pages/Users/components/CreateForm/utils.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+const alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+const digits = '0123456789';
+
+// Note: password requires a minimum of 8 characters and contains a mix of letters, numbers, and symbols.
+export const createInitialPassword = () => {
+  const randomAlphabet = customAlphabet(alphabet, 1)();
+  const randomDigit = customAlphabet(digits, 1)();
+
+  return `${randomAlphabet}${randomDigit}${nanoid(6)}`;
```
Comment (gao-sun) [inline packages/console/src/pages/Users/components/CreateForm/utils.ts:11]: how about replace `nanoid()` with 6 random (alphabet or digits) to be user friendly for copy-paste?

**Pull request merged**